tests/js/navIconsTests.js

```diff
@@ -0,0 +1,96 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+    "use strict";
+
+    fluid.registerNamespace("gpii.tests");
+
+    gpii.tests.hasClass = function (elementName, element, selector, expected) {
```

Comment:
This seems to be duplicated from the other test. Maybe abstract these out into a utility file for the tests and use a `gpii.tests.utils` namespace.